packages/chart/src/components/BarChart.jsx

```diff
@@ -21,7 +21,17 @@ const BarChart = ({ xScale, yScale, seriesScale, xMax, yMax, handleTooltipMouseO
         <BarChartType.Horizontal xScale={xScale} yScale={yScale} xMax={xMax} yMax={yMax} seriesScale={seriesScale} />
 
         {/* tooltips */}
-        <Bar key={'bars'} width={Number(xMax)} height={Number(yMax)} fill={false ? 'red' : 'transparent'} fillOpacity={0.05} onMouseMove={e => handleTooltipMouseOver(e, data)} onMouseOut={handleTooltipMouseOff} onClick={e => handleTooltipClick(e, data)} />
+        <Bar
+          key={'bars'}
+          display={config.tooltips.singleSeries ? 'none' : 'block'}
+          width={Number(xMax)}
+          height={Number(yMax)}
+          fill={false ? 'red' : 'transparent'}
```

Comment:
Remove the false statement here. It might have already been there, but its not needed so either add in debugging logic or leave it as transparent.